docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -19,6 +19,7 @@ over the place. These will be very challenging to clean up and can eat up a sign
 pile up indefinitely. Suggested time to live time is 30 days.
 
 ### Backups
+#### (This will be Deprecated from version 7.2.1.Please refer Snapshot for backup process)
```

Comment:
from version 0.7.2.1. 
Change the second part to: "Please refer to the Snapshot section for the new way to do backups"

Reply:
this comment still stands

Reply:
Done